R1 reduce jargon & repitition (#297)
SORTEE-Github-Hackathon/manuscript: content/04.use-cases-b.md

<!--## GitHub in EcoEvo examples (Part 2)-->

### Hosting a website{#hosting-a-website}

<!--*Contributors to this section: Rob Crystal-Ornelas, Emma Hudgins*   -->
Personal or laboratory websites can improve the sharing of research findings, build online presence, and increase coordination of research efforts [@doi:10.1038/nj7142-347a].
Despite researchers in ecology and evolution generally lacking experience in building or hosting webpages, many tools have been developed to help this process. 
Simple websites can consist of a single HTML or markdown document that will be rendered as a website by activating GitHub's Pages (<https://pages.github.com>) feature. 

> More complex static websites can now be easily built from a series of markdown documents using a variety of software including literate programming tools like Quarto (<https://quarto.org>) and R packages such as `distill` (<https://github.com/rstudio/distill>) and `blogdown` (<https://bookdown.org/yihui/blogdown>).

Reviewer: Maybe remove sentence since:
1) want to cut down on length
2) blogdown _is_ more technical and prob out of scope for this paper

> The resulting files from static websites can then be hosted in repositories, from which one can activate GitHub's Pages (<https://pages.github.com>) feature to render fully featured and styled websites.

Reviewer: If we remove previous sentence remove this as well.

A CC BY 4.0 license specifies that any code (or other creative products) must be appropriately credited to its original author when distributed, adapted or reused.

### Collaborative and asynchronous code editing{#code-editing}

<!--*Contributors to this section: Kaitlyn Gaynor, Rob Crystal-Ornelas, Ali, Allison Binley*-->

> <!-- I think I accidentally added more jargon, but I feel like this section is really important to get right. I'd be open to leaving all the GitHub jargon out and simply citing a source on this instead.  -->

Reviewer 2: great work, Eric! In the first sentence, I would change "share in-progress code" to "share in-progress work", to remove the focus on code. Similarly, in Line 52, maybe say "with the click of a button, greatly facilitating peer review" or add an "for example"

> In academic settings, GitHub can also facilitate interactions between research advisors and advisees, or among collaborators, providing a platform for everyone to share in-progress code, and flag specific challenges or questions for their supervisors, mentors, or collaborators ([Table 2](#tbl:roles)).

Reviewer: Making more concise. 

```suggestion
In academic settings, GitHub can serve as a platform for everyone (advisor, advisees, collaborators) to share in-progress code, and flag specific challenges or questions for each other ([Table 2](#tbl:roles)).
```

> Periodic code review [@doi:10.1145/3341525.3387370] can also help identify errors early in the scientific process, and inform further training and mentorship to fill gaps in skills.

Reviewer: ```suggestion
Periodic code review [@doi:10.1145/3341525.3387370] is useful for identifying errors early in the scientific process, and informing further training and mentorship to fill gaps in skills.
```

> This is facilitated by a group of core features of git and GitHub that allow contributors to work simultaneously on the same project without interferring with eachother's work.

Reviewer: ```suggestion
This is facilitated by a group of core features of git and GitHub ("branching" and "forking" [Box 1](#definitions)) that allow contributors to work simultaneously on the same project without interfering with each other's work.
```

> "Branching" and "forking" (see [Box 1](#definitions)) allow users to work independently on a linked copy of a repository and then later merge their changes into the main project. 

Reviewer: Removed this sentence becuase I incorporated content into previous sentence
```suggestion
```

Author: I'm not seeing the edit to the previous sentence, but I like where this is going.

> To signal that changes are ready to be merged, one makes a "pull request" (see [Box 1](#definitions)).

Reviewer 3: I like this section and think it highlights one of the major benefits of using GitHub in scientific research. One thing that may be helpful to touch on, however, is the frequency of when you want to be creating pull requests (or in other words what is the link between branching/forking and pull requests?). Something like "After a team member reaches a new milestone, or addresses issues in code that is ready to be merged one can make a "pull request"... Not sure if it's worth it but we could also add a sentence that discusses something such as "GitHub facilitates team members to frequently branch, commit, and merge code so that large projects avoid spending large amounts of time making software environments, functions, and other portions of their code compatible with other team members when products need to be finished for publication" This would also help to address some of the reviewers comments about how investing in time on the front end can avoid long term issues when code needs to be synchronized for final products. Let me know what you think

Reviewer: I don't know, we mention many frequent commits in previous sentence. And I think it's fine to just talk generally within this paragraph about the different git commands like branch, fork, pull without going into "best practices". If we want to include those could put something in the definition table.

Author: I see what you're saying @helenweierbach.  I think our readers would probably find some general guidelines useful (like PR == new milestone), but I'm reluctant to make specific recommendations since I think this stuff depends a lot on the group culture.  How about this?
```suggestion
When a set of related changes contributing some new feature or milestone are ready to be evaluated and merged, one makes a "pull request" (see [Box 1](#definitions)).
```
